src/app/authentication/pages/login/login.component.scss

```diff
@@ -0,0 +1,11 @@
+.content-wrapper {
+  
```

Comment:
@carryall Could we remove this extra space?

Reply:
@junan Same as [this comment](https://github.com/carryall/survey-challenge/pull/23#discussion_r530085962), I'm not sure that we should remove it since it's generated from angular when we generate a new component. If we remove it we need to keep removing it from every component we generate, what do you think?